src/tests/system/tests/test_authentication.py

```diff
@@ -15,55 +15,60 @@
 
 @pytest.mark.topology(KnownTopologyGroup.AnyProvider)
 @pytest.mark.parametrize("method", ["su", "ssh"])
-def test_authentication__using_a_good_then_bad_password(client: Client, provider: GenericProvider, method: str):
+@pytest.mark.importance("critical")
+def test_authentication__with_default_settings(
+    client: Client,
+    provider: GenericProvider,
+    method: str,
+):
     """
-    :title: SSH and su authentication
+    :title: Authenticate with default settings
     :setup:
-        1. Add user to SSSD
-        2. Set password for user
-        3. Start SSSD
+        1. Create user
+        2. Start SSSD
     :steps:
         1. Authenticate user with correct password
         2. Authenticate user with incorrect password
     :expectedresults:
-        1. User is authenticated
-        2. User is not authenticated
+        1. Authentication is successful
```

Comment:
More descriptive sentence might be nice

1. Authentication with correct password is successful
2. Authentication with incorrect password fails

Reply:
I think it's okay. If you look at it in parallel, there is less redundancy, and that was the motive, trim the language. 

1. Authenticate user with correct password
1. Authentication is successful

vs. 

1. Authenticate user with correct password
1. Authentication with correct password is successful

Are you okay with leaving it as is?

Reply:
I forgot that this is cherry-picking too. So I agree, it is better to have them the same.